lib/relaton/bib/address.rb

```diff
@@ -31,6 +31,12 @@ def initialize(**args) # rubocop:disable Metrics/CyclomaticComplexity
         @formatted_address  = args[:formatted_address] unless args[:city] && args[:country]
       end
 
+      def ==(other) # rubocop:disable Metrics/AbcSize
```

Comment:
Lutaml::Model supports equivalence checks of attributes
* https://github.com/lutaml/lutaml-model#comparison